pkgs/development/python-modules/pijuice/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pythonOlder,
```

Comment:
A formatting advice: put each parameter in one line.

And another, more mandatory: the lines should not end with `,`.

Reply:
Done